controllers/novels.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../models')
+
+const getAllNovels = async (request, response) => {
+  try {
+    const novels = await models.novels.findAll({
+      include: [{ model: models.authors }, { model: models.genres }]
+    })
+
+    return response.send(novels)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve the novel, try again')
+  }
+}
```

Comment:
This route doesn't work. It's because you named your linking table `booktype`, Sequelize is very picky about pluralization as I mentioned in class so it is looking for `booktypes`.